__tests__/app.test.js

```diff
@@ -50,3 +50,35 @@ describe("GET categories", () => {
       });
   });
 });
+
+describe("GET review by id", () => {
+  test("200: should respond with an object that contains the relavant review properties", () => {
+    return request(app)
+      .get("/api/reviews/1")
+      .expect(200)
+      .then((res) => {
+        expect(typeof res.body).toBe("object");
+        expect(res.body.reviewToSend).toEqual(
+          expect.objectContaining({
+            review_id: expect.any(Number),
+            title: expect.any(String),
+            review_body: expect.any(String),
+            designer: expect.any(String),
+            review_img_url: expect.any(String),
+            votes: expect.any(Number),
+            category: expect.any(String),
+            owner: expect.any(String),
+            created_at: expect.any(String),
+          })
+        );
+      });
+  });
+  test("404: should return 404 error when an id is requested that does not exist", () => {
+    return request(app)
+      .get("/api/reviews/10000")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body).toEqual({ msg: "Page/File Not Found" });
+      });
+  });
+});
```

Comment:
Have a think about another test which deals with an 'invalid' review id, e.g. /api/reviews/NotAnId

This should return a error 400, please implement this test and logic.